config/credentials.yml.enc

```diff
@@ -1 +1 @@
-MbsKUanbtQDtuCKsAHQ5V5FJA0IacR0cLf9uzMj+1cStvH0oELLLVMUUWGwEBso7vG1/Hl/KtWsymy4yw5KwQEWIknbKNdljjqx6PZAXYN7J/TxcjKjQXn6O59pK5YVcNBrTWY/bWK1+wyZlLki7kAZoyAXyJvGJoqqpV4+oHDzf4TO6Bnzz8qzsQQ9oaBgJ0p33PDjMrCENi5WL1uYQb40xCyBDwRmbOG+3R1HJbK1hTg7fI9eY1177YXAHdzD+LIfsoasdw6Ww/ERNcbhA1o/JOacQ8ys0jt+3YeFyvcYvdFtsCciVOj2s7uAHGmCtF/6Zf3BDr+PEDVAFaBojj8rJOxyAl7jrTox0aoBkgyGfV/+Y33eZqwYEpadKVd4HxpgnLUQ0luNV6UTC95pSrjSFedMA+53815iqG2qzLDrWT0oNtd4AT0xSQ6XYIiCCZhlQPfjwERWacr8a2EHSVmYbgO6UA3CuUTOALnG2zFiGp8hnrmz2/0LwLsBjWVlEfV4bh241P6lLAfc9r1qw3v6uH1hjKyeAjFPmCLxC--lUFKuoQeJwdD/Fgs--avdMql2bOrXtmxgfwmkztg==
\ No newline at end of file
+TnJS88C8DYLQ62uZUfsKOYndeR4WPSyh7KEFJfl7Q4sWnkzALdbDf6bwN9KB2T6Umf3MQBtj45EidUcmUYRxS83PrUQnNsJ1lWCOlx6ccmXSLfXunOegMaDMa1vAIYZH5x93APCUPLr7VdtDJmPOBBq3BvdIvmSAeWGcYLZ0C5PG8txk3B4kqRMaytv3SaikK2S43QNgR/lZxAXW/qBhXpi6l9UxoESTqfSGhBFcnfM8CGI1noOg9HDPOIWqijW0y01FyPKqSuWoKbKnrCFhDb3YQUlL1W6hP0qIDEddEa1341oMPvcZC72RcM2NIjdBiH3tJMRI+8Krtvf0fd/qeXa0bKowAKeT78VG1n5Xnii2WlmCPNx3RgGmApjA/c5ZrqvnK2jOlyjOIRrgfDy3oCLbUASuBRlPB0neV0pNxBPO68kk7OJ9VAj5dX/QTumb9bqGmSschWNn8z8sMNr4XE5xwnHu4IfYFR7RCrxPHe2Uv1Wir/BW/a1jzY5HaMwVgMMq15vy+K+oncZUlvydLmXASuKnFor2l2/FvI+7--i6QLDmGeEGUh7OKx--e2j8Mmz5ypP3TCIrfD4GhQ==
```

Comment:
Why do you need to update this?

Reply:
To have a functioning callback URL in development. Since I'm using NGROK, everytime I close and reopen it, a new URL is generated and I need to update credentials as well.